.github/workflows/heroku-remove-app.yml

```diff
@@ -0,0 +1,29 @@
+name: Heroku Pull Request
```

Comment:
Excuse my lack of gitflow knowledge, where do these workflows get called? is that just setup in git? Or does it just know to run this workflow whenever there is a "closed" git action?

Reply:
they're executed but github automatically. github looks in the `.github/workflows` directory for scripts to register/run. so yeah, your intuition is correct -- when the 'closed' action happens, github checks for a workflow triggered by that event.